@here/harp-examples/src/rendering_synchronous.ts

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { GeoCoordinates } from "@here/harp-geoutils";
+import { CopyrightElementHandler, MapView, MapViewEventNames } from "@here/harp-mapview";
+import { APIFormat, OmvDataSource } from "@here/harp-omv-datasource";
+import THREE = require("three");
+import { accessToken } from "../config";
+
+export namespace SynchronousRendering {
```

Comment:
Please add proper documentation what this example is showing and how it is achieved using typedoc snippets, like it is done for the hello world example:
https://github.com/heremaps/harp.gl/blob/master/%40here/harp-examples/src/getting-started_hello-world_npm.ts
https://www.harp.gl/docs/master/doc/modules/harp_examples.helloworldexample.html
The documentation is generated with:
`yarn typedoc`